package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "stacked-sheet",
+  "main": "src/stacked-sheet.js",
+  "module": "src/stacked-sheet.js",
+  "files": [
+    "src"
+  ],
+  "version": "0.0.0",
+  "description": "Webcomponent stacked-sheet with lit-element and open-wc recommendations",
+  "license": "MIT",
+  "dependencies": {
+    "lit-element": "^2.0.1"
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "prettier --write",
+      "git add"
+    ]
+  },
+  "scripts": {
+    "lint": "npm run lint:eslint && npm run lint:prettier",
+    "format": "npm run format:eslint && npm run format:prettier",
+    "lint:eslint": "eslint --ext .js,.html .",
+    "format:eslint": "eslint --ext .js,.html . --fix",
+    "lint:prettier": "prettier \"**/*.js\" --list-different || (echo '↑↑ these files are not prettier formatted ↑↑' && exit 1)",
+    "format:prettier": "prettier \"**/*.js\" --write",
+    "build": "del-cli dist && rollup -c rollup.config.js",
+    "demo": "del-cli demo && rollup -w -c rollup.config.js --demo",
+    "test": "karma start karma.config.js --coverage",
+    "test:watch": "karma start karma.config.js --auto-watch=true --single-run=false",
+    "test:es5": "karma start karma.config.js --legacy --coverage",
+    "test:es5:watch": "karma start karma.config.js --legacy --auto-watch=true --single-run=false"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.3.4",
+    "@babel/plugin-proposal-class-properties": "^7.3.4",
+    "@babel/plugin-proposal-decorators": "^7.3.0",
+    "@babel/preset-env": "^7.3.1",
+    "@commitlint/cli": "^7.5.2",
+    "@commitlint/config-conventional": "^7.5.0",
+    "@open-wc/eslint-config": "^0.4.3",
+    "@open-wc/prettier-config": "^0.1.10",
+    "@open-wc/testing": "^0.11.4",
+    "@open-wc/testing-karma": "^1.1.1",
+    "@webcomponents/webcomponentsjs": "^2.2.10",
+    "fast-async": "^6.3.8",
+    "less": "^3.9.0",
```

Comment:
I think we don't need this package anymore. As well as `@babel/plugin-proposal-class-properties`, `@babel/plugin-proposal-decorators`, `postcss-url` and `rollup-plugin-postcss`.

Reply:
yup, agreed. All gone now :)